content/projects/gsoc/2020/projects/winsw-yaml-configs.adoc

```diff
@@ -63,18 +62,32 @@ Usage of YAML should simplify configuration management in Jenkins, especially wh
 
 During each phases respective unit test will be added to existing Unit Test library.
 
-=== Future Improvements
-* Complete developer guide for YAML support service descriptor.
-* Complete user documentation for Config validations. + 
+== Current Status
+https://github.com/winsw/winsw/releases/tag/v2.10.0[MVP has been released.] This release contains following features.
 
-If time allows both will be delivered in Phase 3
+* YAML configuration parsing
+* YAML configuration support for extensions
+* https://github.com/winsw/winsw/blob/master/doc/yamlConfigFile.md[YAML configuration documentation]
+* https://github.com/winsw/winsw/blob/master/examples/sample-allOption.yml[Sample YAML configuration file]
+* Unit test for YAML configuration support
 
-=== Articles
+== Ongoing improvements
+* YAML schema validation
+** JSON schema file has been created. https://github.com/winsw/winsw/pull/656[Pull request]
+
+== Further Improvements
+* Configuration validation on Startup
+** Environment variable expansion and validation is left to do.
+
+== Articles
 * link:/blog/2020/07/2020-07-08-winsw-yaml-support[GSoC 2020 - Phase 01 Updates]
 * https://medium.com/runtimeerror/path-to-gsoc-2021-e6a81e62308d[Path to GSoC2021 @ *Medium*]
 
+== Demos
+* https://www.youtube.com/watch?v=9qyo1f2rKQw&feature=youtu.be[Phase 1 Demo]
+* https://www.youtube.com/watch?v=9qyo1f2rKQw[Phase 2 Demo]
 
-=== Office hours
+== Reach us
```

Comment:
```suggestion
== Contact us
```

Taking the date, you can safely remove the office hours reference. I'd guess we will revisit the format and time with @NextTurn later